app/src/main/java/com/example/nsc_events/model/Event.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.nsc_events.model
+
+data class Event(
+    val eventTitle: String,
+    val eventDescription: String,
+    /* TODO: change date variable to a Date data type instead of string */
+    val eventDate: String,
+    /* TODO: what other event properties should be added? */
```

Comment:
Perhaps end time? Event Host? So is it student body leadership hosting, or a club, or a schoolwide event? Is the event a paid event or a free event? Just spitballing to see what pops up.